src/driver/sqlite-abstract/AbstractSqliteQueryRunner.ts

```diff
@@ -1197,4 +1218,18 @@ export abstract class AbstractSqliteQueryRunner extends BaseQueryRunner implemen
         this.replaceCachedTable(oldTable, newTable);
     }
 
+    /**
+     * tablePath e.g. "myDB.myTable", "myTable"
+     */
+    protected splitTablePath(tablePath: string): [string | undefined, string] {
+        return ((tablePath.indexOf('.') !== -1) ? tablePath.split('.') : [undefined, tablePath]) as [string | undefined, string]
```

Comment:
Should be careful here, `split('.')` could return more than 2. Better to do something like
```suggestion
        let dotIndex = tablePath.indexOf(".");
        return (dotIndex >= 0) ? [tablePath.substr(0, dotIndex), tablePath.substr(dotIndex + 1)] : [undefined, tablePath];
```

Reply:
Also make sure all strings are in double quotes rather than single, should be picked up by ESLint in IDE?